components/skus/browser/rs/lib/src/sdk/credentials/present.rs

```diff
@@ -61,6 +61,50 @@ where
                 let name = "sku#".to_string() + &item.sku;
 
                 let (expires_at, presentation) = match item.credential_type {
+                    CredentialType::TimeLimitedV2 => {
+                        let tlv2_cred = self
+                            .matching_time_limited_v2_credential(&item.id) // get tokens for today
+                            .await?
+                            .ok_or(InternalError::ItemCredentialsMissing)?;
+                        let unblinded_creds =
+                            tlv2_cred.unblinded_creds.ok_or(InternalError::ItemCredentialsMissing)?;
+
+                        // retrieve the next unspent token
+                        let (i, cred) = unblinded_creds
+                            .into_iter()
+                            .enumerate()
+                            .find(|(_i, cred)| !cred.spent)
+                            .ok_or(InternalError::OutOfCredentials)?;
+
+                        let issuer = self.encode_issuer_id(&order.merchant_id, &item.sku)?;
+
+                        let verification_key =
+                            cred.unblinded_cred.derive_verification_key::<Sha512>();
+                        // FIXME change the payload we're creating the binding with
```

Comment:
double checking this is intended as a followup

Reply:
https://github.com/brave/brave-core/pull/15465/commits/9c9188c99dcc3e8bb08fd4835f602a62155975ae